src/pages/gen2/start/quickstart/vite-react-app/index.mdx

```diff
@@ -0,0 +1,270 @@
+export const meta = {
+  title: 'Vite + React App',
+  description: 'Get started with AWS Amplify (Gen 2) using React + Vite.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Vite and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Vite](https://vitejs.dev/guide/), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
```

Comment:
```suggestion
This Quickstart guide will walk you through how to build a task list application with TypeScript, Vite, and React. If you are new to these technologies, we recommend you go through the [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html), [Vite](https://vitejs.dev/guide/), and official [React](https://react.dev/learn/tutorial-tic-tac-toe) tutorials first.
```

+oxford comma
reorder tutorial links in the order they appear in the preceding sentence (just noticed the other guides are different, but I think it reads easier)